spec/views/post_show_spec.rb

```diff
@@ -0,0 +1,74 @@
+require 'rails_helper'
+RSpec.describe 'Post show page', type: :feature do
+  before(:each) do
+    @user1 = User.create(
+      name: 'Alejandra',
+      bio: 'Aspiring FullStack Dev',
+      photo: 'https://unsplash.com/photos/iR3dtvKmwAw',
+      posts_counter: 2
+    )
+
+    @user2 = User.create(
+      name: 'Christopher',
+      bio: 'FullStack Dev',
+      photo: 'https://unsplash.com/photos/hodKTZow_Kk',
+      posts_counter: 3
+    )
+
+    @post1 = Post.create(
+      author_id: @user1.id,
+      title: 'Test',
+      text: 'First Post',
+      comments_counter: 1,
+      likes_counter: 1
+    )
+
+    @comment1 = Comment.create(
+      text: 'First Comment',
+      post_id: @post1.id,
+      user_id: @user1.id
+    )
+
+    @like1 = Like.create(
+      post_id: @post1.id,
+      user_id: @user1.id
+    )
+  end
+
+  describe 'the post show page' do
+    it 'displays the post title' do
+      visit user_post_path(@user1, @post1)
+      expect(page).to have_content('show post for a given user')
+    end
+
+    it 'displays the post author' do
+      visit user_post_path(@user1, @post1)
+      expect(page).to have_content('Alejandra')
+    end
+
+    it 'displays number of comments' do
+      visit user_post_path(@user1, @post1)
+      expect(page).to have_content('show post for a given user')
+    end
+
+    it 'displays number of likes' do
+      visit user_post_path(@user1, @post1)
+      expect(page).to have_content('show post for a given user')
+    end
```

Comment:
- [ ] The tests that assert that you can see how many comments and likes a post has are testing what is expected of you.
You are currently expecting page to `have_content('show post for a given user')`. Please modify these tests 👍